### PR TITLE
Added PSD Var and Finalize-events-template-rate

### PR DESCRIPTION
**Pull request by KoustavChandra**
Updated the `pycbc_inspiral_skymax` to support the psd-variation statistics and finalize-events-template-rate.

### CODE REVIEW

File: bin/pycbc_inspiral_skymax [hunk 1]
```diff
@@ -210,7 +224,22 @@ with ctx:
...
+    if opt.psdvar_segment is not None:
+        logging.info("Calculating PSD variation")
+        psd_var = pycbc.psd.calc_filt_psd_variation(gwstrain, opt.psdvar_segment,
+                opt.psdvar_short_segment, opt.psdvar_long_segment,
+                opt.psdvar_psd_duration, opt.psdvar_psd_stride,
+                opt.psd_estimation, opt.psdvar_low_freq, opt.psdvar_high_freq)
+
+    if opt.enable_q_transform:
+        logging.info("Performing q-transform on analysis segments")
+        q_trans = qtransform.inspiral_qtransform_generator(segments)
```
Comment (spxiwh) [inline bin/pycbc_inspiral_skymax:237]: As far as I can see this function doesn't exist??

**Review by spxiwh — APPROVED** [review]
Hi Koustav. Thanks for doing this. It's approved for merging, but please check out the qtransform thing. I can see that it's in pycbc_inspiral as well, but it doesn't seem to exist in the actual code base!

An open question here, there are now 3 `pycbc_inspiral` like codes that are all very similar (`pycbc_multi_inspiral` which should do a coherent search is the other). Problems like this one where pycbc_inspiral gets updated and the others don't are common. We need to consider how to avoid this.

File: bin/pycbc_inspiral_skymax (hunk 1 above)
Comment (ahnitz) [reply bin/pycbc_inspiral_skymax:237]: Looks like it was remove some time ago, probably my mistake for not catching to remove it here as well. I don't remember the history here.

Comment (spxiwh) [reply bin/pycbc_inspiral_skymax:237]: https://github.com/gwastro/pycbc/blob/master/bin/pycbc_inspiral#L291

I don't think this was removed from the main pycbc_inspiral.

Comment (spxiwh) [reply bin/pycbc_inspiral_skymax:237]: (Maybe that's what you meant, and I'm just a bit confused this morning :-) )

Comment (ahnitz) [reply bin/pycbc_inspiral_skymax:237]: @spxiwh No, I agree, it seems to have been left in the exes. I think the code it relies upon was removed though at some point. The underlying code used to exists, it was most likely removed in a shuffling by me, but I don't really remember what the thought process was then. At this point though, these lines are dead code and should just be removed.

**Pull request merged**